examples/with-fingerprintjs-pro/pages/index.tsx

```diff
@@ -0,0 +1,13 @@
+import type { NextPage } from 'next'
```

Comment:
I wonder why this page is needed?

Reply:
Thanks, fixed! Didn't know about `redirects` section for `next.config.js` before.